File: Ej 17 (En Mayuscula)/Kotlin/EnMayuscula.kt
//DeathbatO

import java.util.Scanner;

fun mayus(frase: String): String {
    val fraseMayus = StringBuilder()
    val palabras = frase.split(" ")

    for (palabra in palabras) {
        if (palabra.isNotEmpty()) {
            fraseMayus.append(palabra[0].toUpperCase() + palabra.substring(1))
            fraseMayus.append(" ")
        }
    }
    return fraseMayus.toString().trim()
}

fun main() {
    val scanner = Scanner(System.`in`)
    print("Por favor ingrese una frase: ")
    val frase = scanner.nextLine()
    println("La frase con mayúsculas corregidas:")
    println(mayus(frase))
    scanner.close()
}

File: Ej 17 (En Mayuscula)/Python/EnMayuscula.py
## DeathbatO

def mayus(frase):
    frase_mayus = ""
    palabras = frase.split(" ")
    
    for palabra in palabras:
        if len(palabra) > 0:
            palabra = palabra[0].upper() + palabra[1:]
        frase_mayus += palabra + " "
    
    return frase_mayus.rstrip()

frase = input("Por favor ingrese una frase: ")
print("La frase con mayúsculas corregidas:")
print(mayus(frase))
